synfig-studio/src/gui/pluginmanager.h

```diff
@@ -127,6 +129,7 @@ class Plugin
 	PluginString description;
 
 	bool is_valid() const;
+	void launch_dir() const;
```

Comment:
I don't think that PluginManager class should have this method. IMO it should be done in Plugin Manager Dialog